### DISCUSSION TITLE
Weekly Governance Updates (24 February, 2023)

### DISCUSSION THREAD

**Discussion by luckyparadise** [filecoin-project/FIPs#643] [General]
This weekly report provides a high-level overview of what's happening with Filecoin governance.

Anyone is welcome to participate in governance. To learn more about any of the below materials, simply follow the links and feel free to weigh in.

If you have questions or ideas related to FIPs or Filecoin governance, please start a new discussion in the FIPs Forum or post directly in [#fil-fips](https://github.com/filecoin-project/FIPs/discussions/filecoinproject.slack.com).

---

**Accepted FIPs** - *FIPs awaiting network implementation*

- FIP0047 - Proof Expiration & PoRep Security Policy [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0047.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/415)
- FIP0048 - f4 Address Class [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0048.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/473)
- FIP0049 - Actor Events [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0049.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/484)
- FIP0050 - API between user-programmed actors and built-in actors [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0050.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/401)
-  FIP0052 - Increase max sector commitment to 3.5 years [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0052.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/475)
- FIP0054 - Filecoin EVM Runtime (FEVM) [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0054.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/592)
- FIP0055 - Supporting Ethereum Accounts, Addresses, and Transactions [(FIP)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0055.md)[(Discussion 1)](https://github.com/filecoin-project/FIPs/discussions/388)[(Discussion 2)](https://github.com/filecoin-project/FIPs/discussions/490)[(Discussion 3)](https://github.com/filecoin-project/FIPs/discussions/287)
- FIP0057 - Update Gas Charging Schedule and System Limits for FEVM [(FIP)]([0057](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0057.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/588)

**Last Call** - *Final opportunity for community feedback!*
NA

**Current FIP Drafts** - *Now accepting community feedback; merged into repo*

- FIP0037 - Gas Model Adjustment for User Programmability [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0037.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/316)
- FIP0039 - Filecoin Message Replay Protection [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0039.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/301)
- FIP0056 - Sector Duration Multiplier [(FIP Draft)](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0056.md)[(Discussion)](https://github.com/filecoin-project/FIPs/discussions/554)

 **:fire: HOT :fire: Discussion Topics**

- @ribasushi & @f8-ptrk started a pre-FIP discussion on a Capped Duration Multiplier (CDM) Policy. Capped Duration Multiplier (CDM) policy is introduced for all new and existing sectors on the filecoin network. A sector committed for a longer time receives progressively higher Quality Adjusted Power, compared to a sector with shorter commitment, all else being equal. Please read more about this discussion [here](https://github.com/filecoin-project/FIPs/discussions/636)
- @anorth @Kubuxu @ZenGround0 have done some work and thinking around built-in market cron risk and mitigations. Read more [here](https://github.com/filecoin-project/FIPs/discussions/638)

**Other Governance Updates**
* Plans are rapidly unfolding for nv18 - to stay in the loop, be sure to check out the [planning thread ](https://github.com/filecoin-project/community/discussions/74?sort=new#discussioncomment-4912995)!

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
hey, i am co-author of the CDM FIP and have not done work on the market cron risks :) 

@f8-ptrk thanks for flagging this. it has been resolved now.
